### PR TITLE
better structure for docker-container and removed become:yes in some …

### PR DESCRIPTION
**Pull request by ToivoHn**
…places

### CODE REVIEW

File: ansible/inventory/local/group_vars/docker-deploy.yml [hunk 1]
```diff
@@ -14,3 +14,4 @@ shortcut_name: "Shadow_Hand_Launcher" # Specify the name for the desktop icon (d
 optoforce: false # Specify if optoforce sensors are going to be used with a branch name (Only for Flexible Hand)
 launch_hand: false # Specify if hand driver should start when double clicking desktop icon
 demo_icons: true # Generates desktop icons to run demos
+run_when_docker_launches: "terminator -x bash -c '/usr/local/bin/setup_dexterous_hand.sh && bash || bash'" #what to run when the Docker launches
```
Comment (AndriyPt) [inline ansible/inventory/local/group_vars/docker-deploy.yml]: no commands in ansible parameters. Find more elegant way please :)

Comment (ToivoHn) [reply ansible/inventory/local/group_vars/docker-deploy.yml]: Fixed in the latest commit

File: ansible/inventory/teleop/group_vars/server.yml [hunk 2]
```diff
@@ -7,3 +7,5 @@ nvidia_docker: false # Enable nvidia docker
 cyberglove: "A-penny" # Specify the branch of sr_cyberglove_config for cyberglove configuration
 use_aws: false # Flag to prompt for customer key for uploading files to AWS
 use_openvpn: false # Specify if openvpn-server should be installed
+setup_directory: "/usr/local/bin/"
+setup_script: "teleop-server-setup.sh"
```
Comment (AndriyPt) [inline ansible/inventory/teleop/group_vars/server.yml]: setup script should be hardcoded in task not as param

Comment (ToivoHn) [reply ansible/inventory/teleop/group_vars/server.yml]: Fixed in the last commit

**Pull request merged**